src/witan/sen2/sen2.clj

```diff
@@ -0,0 +1,24 @@
+(ns witan.sen2.sen2
+  (:require [clojure.java.io :as io]
+            [tablecloth.api :as tc]))
+
+"A script which pulls data for a single gss-code from the SEN2 caseload, and generates total SEN2 counts per LAn"
+
+(def caseload-2010-2022 (delay (-> "sen2_estab_caseload.csv"
+                                   io/resource
+                                   io/file
+                                   tc/dataset))) ;; this doesn't work with districts
```

Comment:
should the header be keywordised?

Reply:
just change `(tc/dataset)` to `(tc/dataset {:key-fn keyword})`

Reply:
more details here: https://scicloj.github.io/tablecloth/index.html#Dataset_creation